ui/components/badge.js

```diff
@@ -2,16 +2,18 @@ import { XIcon } from '@heroicons/react/outline'
 
 export default function Badge({ children, onRemove }) {
   return (
-    <div className='my-1 mr-1 flex items-center justify-center overflow-hidden text-ellipsis rounded-md bg-gray-800 py-1 px-2 font-medium text-white'>
-      <div className='max-w-full flex-initial overflow-hidden text-ellipsis text-xs font-normal leading-none'>
+    <div className='my-1 mr-1 flex items-center justify-center overflow-hidden text-ellipsis rounded-md bg-gray-800 font-medium text-white first:my-0'>
```

Comment:
to prevent input changes size between being empty and having 1+ badges

Reply:
nice!